js/_answerToggle.js

```diff
@@ -0,0 +1,19 @@
+export default function answerToggle() {
+  const questionCards = document.querySelectorAll("[data-js=questionCards]");
+  questionCards.forEach((questionCard) => {
+    const showAnswerButton = questionCard.querySelector(
+      "[data-js=showAnswerButton]"
+    );
+    const answer = questionCard.querySelector("[data-js=answer]");
+    showAnswerButton.addEventListener("click", () => {
+      answer.classList.toggle("hidden");
+      if (answer.classList.includes("hidden") === true) {
```

Comment:
You already fixed the code for the button but just wanted to say that "(answer.classList.includes("hidden") === true)" is a bit too long. You don't need the === true part.
Writing "if (answer.classList.includes("hidden"))" should be enough since it already checks for it being true.

Reply:
You are right, === true ist not necessary.
It didn't work with .includes but with .contains :)
Thanks for reviewing! :)